src/Finder.php

```diff
@@ -27,6 +27,10 @@ public function __construct()
         $this
             ->files()
             ->name('*.php')
+            ->ignoreDotFiles(false)
```

Comment:
i would consider it a bc break. ppl were asking all the times how to avoid changing .files and they were quite happy that Finder started to ignore them by default

Reply:
No BC, `ignoreDotFiles()` can still be configured by the user now.